includes/MediaWiki/Api/EditEndpoint.php

```diff
@@ -73,16 +72,25 @@ public static function factory(
 	}
 
 	public function run() {
-		if ( !$this->user->isRegistered() ) {
+		// Rest Validator returns null if submitted as form
+		$requestBody = $this->getValidatedBody();
+		if ( $requestBody === null ) {
+			throw new LocalizedHttpException(
+				MessageValue::new( 'wikibasereconcileedit-editendpoint-invalid-reconcile-json' ),
```

Comment:
I don’t think this is the right message, since it’s not really the reconcile JSON, specifically, that’s invalid.

Reply:
done